central/processindicator/store/postgres/gen.go

```diff
@@ -1,3 +0,0 @@
-package postgres
-
-//go:generate pg-table-bindings-wrapper --type=storage.ProcessIndicator --search-category PROCESS_INDICATORS --references storage.Deployment
```

Comment:
add back in with the --schema-only flag

Reply:
For process indicators it's actually not going to work, as the custom modifications are in the schema. Will do that for plop objects.